about.html

```diff
@@ -0,0 +1,163 @@
+    ></script>
+    <link href="https://fonts.googleapis.com/css2?family=Lato:ital,wght@0,100;0,300;0,400;0,700;0,900;1,100;1,300;1,400;1,700;1,900&display=swap"
+      rel="stylesheet"
+    />
+  <link rel="stylesheet" href="about.css">
+  <title>About Burning Circle Flight Academy</title>
+</head>
+<body>
+  
+  <header>
+    <aside class="top-section">
+      <div class="bar-container">
+      <ul class="bars">
+        <li>
+          <i class="fab fa-facebook-f"></i>
+        </li>
+        <li>
+          <i class="fab fa-twitter"></i>
+        </li>
+        <li>
+          <h4>English</h4>
+        </li>
+        <li>
+          <h4>My Page</h4>
+        </li>
+        <li>
+          <h4>Logout</h4>
+        </li>
+      </ul>
+      </div>
+    </aside>
+    <div class="navbar">
+        <div class="logo-DT">
+          <a href="index.html"><img src="images/plane3.svg" alt="plane" class="plane"></a>
+          <div class="menu-bar nav" id="menu">
+            <div class="icon"></div>
+            <div class="icon"></div>
+            <div class="icon"></div>
+          </div>
+          <div class="cancel" id ='close'>
+            <p>&times;</p>
+          </div>
+        </div>
+
+        <nav class="menu-list" id="nav-mobile">
+          <ul class="items">
+            <li class="home">
+              <a href="index.html">Home</a>
+            </li>
+
+            <li class="links">
+              <a href="about.html">About</a>
+            </li>
+            <li class="links">
+              <a href="#">Program</a>
+            </li>
+            <li class="links">
+              <a href="#">Join</a>
+            </li>
+            <li class="links">
+              <a href="#">Sponsor</a>
+            </li>
+            <li class="links">
+              <a href="#">News</a>
+            </li>
+          </ul>
+          <div class="company">
+            <a href="index.html">Burning Circle</a>
+          </div>
+        </nav>
+    </div>
+  </header>
+  <section class="first-section">
+    <div class="first-section-container">
+      <h1 class="open-heading">The Burning Circle Flight Academy</h1>
+      <div class="first-section-para">
+        <p>
+          The Burining Circle Flight Academy airshow is one of Zambia's first airshows 
+          to be hosted by a privately owned aviation institution. The whole essence of 
+          this event is to connect aerospace industry proffessionals across Africa and
+          the world at Large. This is also a great opportunity for youths interested in
+          aviation and becoming Pilots to learn from the best Pilots and know what it takes
+          to become a world class pilot. 
+        </p>
+      </div>
+      <div class="event-details">
+        <p>Please contact us per email for any further questions.
+        
+        </p>
+        <p class="email-address">
+          burningcircleflightacademy@gmail.com
+        </p>
+      </div>
+    </div>
+  </section>
```

Comment:
- [ ] As suggested by the previous code reviewer, it is unnecessary to have the `<div class="first-section-container">` wrapping the elements, because they are already wrapped in the `<section class="first-section">`. You should consider it to keep your HTML semantic.
- [ ] The same happens in the following lines: 105-115, 117-126.

Reply:
Done